lib/src/editor/toolbar/mobile/mobile_toolbar_style.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+/// Style for the mobile toolbar.
+///
+/// foregroundColor -> text and icon color
+///
+/// itemHighlightColor -> selected item border color
+///
+/// itemOutlineColor -> item border color
+class MobileToolbarStyle extends InheritedWidget {
+  final Color backgroundColor;
+  final Color foregroundColor;
+  final Color itemHighlightColor;
+  final Color itemOutlineColor;
+  final double toolbarHeight;
+  final double borderRadius;
+
+  const MobileToolbarStyle({
+    Key? key,
+    required this.backgroundColor,
+    required this.foregroundColor,
+    required this.itemHighlightColor,
+    required this.itemOutlineColor,
+    required this.toolbarHeight,
+    required this.borderRadius,
+    required Widget child,
+  }) : super(key: key, child: child);
+
+  static MobileToolbarStyle of(BuildContext context) {
+    final MobileToolbarStyle? result =
+        context.dependOnInheritedWidgetOfExactType<MobileToolbarStyle>();
+    assert(result != null, 'No MobileToolbarStyle found in context');
+    return result!;
```

Comment:
```suggestion
    return context.dependOnInheritedWidgetOfExactType<MobileToolbarStyle>()!;
```

There's no difference if you force unwrap the value here, although you add an assert.